.circleci/config.yml

```diff
@@ -14,63 +15,134 @@ jobs:
       - TOOL_NODE_FLAGS: "--max-old-space-size=4096"
 
     steps:
-      - setup_remote_docker
       - checkout
 
-      # install OS dependencies
       - restore_cache:
           name: Restoring Meteor cache
-          key: meteor
+          key: meteor-{{ .Branch }}
 
-      - run: .circleci/install.sh
+      # Install Meteor
+      - run: .circleci/install_meteor.sh
 
-      - save_cache:
-          name: Saving Meteor to cache
-          key: meteor
-          paths:
-            - ~/.meteor
+      # install OS dependencies
+      - run: sudo apt-get update
```

Comment:
Added `sudo` here (and a couple other places) because the new node image throws permission denied errors on those commands. If there's a better way to this please point me to it...